### PR TITLE
Revert Feat/components/list/pagination custom icons

### PR DESCRIPTION
**Pull request by sdwvit**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Duplicate icons

**What is the new behavior?**
Remove duplicates

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No, checked

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

File: output/components.eslint.txt [hunk 1]
```diff
@@ -18,6 +18,12 @@ Scoping to packages that match 'react-talend-components'
   2:1  error  '@kadira/react-storybook-addon-info' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   7:2  error  Unexpected require()                                                                                      global-require
 
+/home/travis/build/Talend/ui/packages/components/src/List/Toolbar/Pagination/Pagination.component.js
+  31:19  error  Strings must use singlequote  quotes
```
Comment (jare-talend) [inline output/components.eslint.txt]: plz fix eslint error

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

File: output/components.eslint.txt (hunk 1 above)
Comment (sdwvit) [inline output/components.eslint.txt]: how is it connected to this pr?

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://391.talend.surge.sh)

**Pull request merged**